merlin/systems/workflow/tensorflow.py

```diff
@@ -56,13 +56,27 @@ def _transform_outputs(self, tensors):
                 d = d.reshape(row_dim, col_dim)
                 output_tensors.append((name, d))
             elif isinstance(value, tuple):
-                # convert list values to match TF dataloader
-                values = value[0].astype(self.output_dtypes[name])
```

Comment:
In the case of a Workflow with a ragged output. this output_dtypes contains the keys matching the triton output names e.g. `{col_name}__values`. This was raising a KeyError here in this lookup into output_dtypes.

Added a condition to check the column schema `is_ragged` attribute to conditionally return the ragged outputs